src/databaseTests/src/test/kotlin/org/vaccineimpact/api/databaseTests/tests/ScenarioTests.kt

```diff
@@ -48,6 +51,107 @@ class ScenarioTests : RepositoryTests<ScenarioRepository>()
         }
     }
 
+    @Test
+    fun `getScenariosForTouchstone returns ordered scenarios`()
+    {
+        val disease1 = "disease-abc"
+        val disease2 = "disease-def"
+        withDatabase {
+            setUp(it)
+            setUpDisease(it, disease2)
+            setUpDisease(it, disease1)
+        }
+        withRepo {
+            val scenarios = it.getScenariosForTouchstone(touchstoneVersionId,
+                    ScenarioFilterParameters())
+
+            assertThat(scenarios.count()).isEqualTo(6)
+            assertThat(scenarios[0].description).isEqualTo("none")
+            assertThat(scenarios[0].disease).isEqualTo(disease1)
+            assertThat(scenarios[1].description).isEqualTo("routine")
+            assertThat(scenarios[1].disease).isEqualTo(disease1)
+            assertThat(scenarios[2].description).isEqualTo("campaign")
+            assertThat(scenarios[2].disease).isEqualTo(disease1)
+
+            assertThat(scenarios[3].description).isEqualTo("none")
+            assertThat(scenarios[3].disease).isEqualTo(disease2)
+            assertThat(scenarios[4].description).isEqualTo("routine")
+            assertThat(scenarios[4].disease).isEqualTo(disease2)
+            assertThat(scenarios[5].description).isEqualTo("campaign")
+            assertThat(scenarios[5].disease).isEqualTo(disease2)
+        }
+    }
+
+    @Test
+    fun `getScenariosForTouchstone only returns scenarios for given touchstone`()
+    {
+        val disease1 = "disease-abc"
+        val disease2 = "disease-def"
+        withDatabase {
+            setUp(it)
+            setUpDisease(it, disease2)
+            setUpDisease(it, disease1)
+        }
+        withRepo {
+            val scenarios = it.getScenariosForTouchstone("fake-id",
+                    ScenarioFilterParameters())
+
+            assertThat(scenarios.count()).isEqualTo(0)
+        }
+    }
+
+    @Test
+    fun `getScenariosForTouchstone only returns scenarios for given filter params`()
+    {
+        val disease1 = "disease-abc"
+        val disease2 = "disease-def"
+        withDatabase {
+            setUp(it)
+            setUpDisease(it, disease2)
+            setUpDisease(it, disease1)
+        }
+        withRepo {
+            val scenarios = it.getScenariosForTouchstone("fake-id",
+                    ScenarioFilterParameters(disease = "fake-disease"))
+
+            assertThat(scenarios.count()).isEqualTo(0)
```

Comment:
This test might be a bit stronger if it filtered to one of the real diseases and then checked that a) you got 3 rows b) all three scenarios belonged to right disease.